microsoft/vscode
# Enable word selection with a key modifier.

This PR fixes #23957

The click link to gesture feature can be disabled and instead use word selection which is expected by some users as described in the issue.

Test:
1. Open file and confirm holding Ctrl allows click to link gesture
![image](https://user-images.githubusercontent.com/30425852/128428138-8805c227-fff1-48cd-b1e6-ce086c8ba15e.png)
2. Hold Ctrl and click comment to place cursor
![image](https://user-images.githubusercontent.com/30425852/128428314-73e1cedf-97fd-472e-addd-e027c6eb6a36.png)
3. Hold Alt and click to place multiple cursors
![image](https://user-images.githubusercontent.com/30425852/128428770-6020dbe3-a2eb-4186-9ea4-badf93b20cf6.png)
4. Open settings and enable Word Selection
![image](https://user-images.githubusercontent.com/30425852/128428234-f358172c-c988-4e88-a26e-5662ebbc62ff.png)
5. Hold Ctrl and click the same location from step 1 to select the entire word
![image](https://user-images.githubusercontent.com/30425852/128428461-6b190ae2-378b-4a81-aad8-c4a5591ae7f3.png)
6. Hold Ctrl and click comment to select the entire word
![image](https://user-images.githubusercontent.com/30425852/128429331-5b443584-c4eb-4a94-9186-35c97612005f.png)
7. Same beahviour as step 3
8. Open settings to change the modifier key `"editor.multiCursorModifier": "ctrlCmd"` and remove the wordSelection
![image](https://user-images.githubusercontent.com/30425852/128429002-e48fa3e5-5c46-402e-a50a-0a8aab9cfc0c.png)
9. Repeat steps 1 to 7 swapping Ctrl for Alt and vice versa
